### PR TITLE
2402: appointment request messages schema docs

### PR DESCRIPTION
**Pull request by kpethtel**
## Description of change
Adds schema docs for the new appointment request messages index. 

## Original issue(s)
department-of-veterans-affairs/va-mobile-team#2402

## Things to know about this PR
It's just documentation.

### CODE REVIEW

File: modules/mobile/docs/openapi.yaml [hunk 1]
```diff
@@ -131,6 +131,68 @@ paths:
...
+            may have been tripped.
+        '504':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: A response from an upstream service timed out.
+      security:
+        - Bearer: []
+      summary: appointment_requests/{appointment_request_id}/messages
```
Comment (aherzberg) [inline modules/mobile/docs/openapi.yaml:195]: Looks good. only a small cosmetic thing, for consistency in the html index file, prepend v0/ to the summary to be similar to the other endpoints.

Comment (kpethtel) [reply modules/mobile/docs/openapi.yaml:195]: Good catch. TY.

File: modules/mobile/docs/openapi.yaml [hunk 2]
```diff
@@ -131,6 +131,68 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appeal/{id}
+  /v0/appointment_requests/:appointment_request_id/messages:
```
Comment (aherzberg) [inline modules/mobile/docs/openapi.yaml:134]: one more small cosmetic thing. Other endpoints use {} to denote a variable rather then colon

Comment (kpethtel) [reply modules/mobile/docs/openapi.yaml]: Good eye. Thank you.

**Review by aherzberg — APPROVED** [review]
Looks good!

**Pull request merged**